packages/web-components/fast-components/src/text-field/index.ts

```diff
@@ -21,4 +27,34 @@ import { TextFieldStyles as styles } from "./text-field.styles";
         delegatesFocus: true,
     },
 })
-export class FASTTextField extends TextField {}
+export class FASTTextField extends TextField {
+    /**
+     * The appearance of the element.
+     *
+     * @public
+     * @remarks
+     * HTML Attribute: appearance
+     */
+    @attr
+    public appearance: TextFieldAppearance;
+    public appearanceChanged(
```

Comment:
the `appearanceChanged()` should be marked as `@internal`

Reply:
done - I'll create an issue to run the repo as there are others which don't have this pattern setup either 👍